.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
Would this make more sense as an `ignorePatterns` array in `.eslintrc` instead? It's only the two lines, and both of these files deal with ESLint configuration.

Reply:
Done. Also seems that it implicitly ignores node_modules
https://eslint.org/docs/latest/user-guide/configuring/ignoring-code